src/main/java/com/configcat/FormattableLogMessage.java

```diff
@@ -0,0 +1,26 @@
+package com.configcat;
+
+public class FormattableLogMessage {
```

Comment:
Since instances of this class are not exposed to end users (apart from the log filter callback), probably it would be better to make it internal.

```suggestion
class FormattableLogMessage {
```